### PR TITLE
Add 1.20 support

### PR DESCRIPTION
**Pull request by Dreeam-qwq**

### CODE REVIEW

**Comment by Jens-Co** [comment]
Tested and working, thanks for your pr!

**Comment by Belialneutral** [comment]
like when they go to update the full GeyserSkinManager to suit the new version of 1.20

File: spigot/pom.xml [hunk 1]
```diff
@@ -91,22 +91,22 @@
             <scope>compile</scope>
         </dependency>
         <dependency>
-            <groupId>com.destroystokyo.paper</groupId>
```
Comment (Jens-Co) [inline spigot/pom.xml:94]: Can you please revert this change? This is not needed and will break pre 1.20 setups.

Comment (Dreeam-qwq) [reply spigot/pom.xml]: Done. Thank you!

Comment (Dreeam-qwq) [reply spigot/pom.xml]: > Can you please revert this change? This is not needed and will break pre 1.20 setups.

Done. Thank you!

**Comment by kuretoshi** [comment]
I tried to run it, but I got an error with paper-1.14.4-245 and could not run it. 

[12:36:25 INFO]: [GeyserSkinManager] Enabling GeyserSkinManager v1.7
[12:36:25 ERROR]: Error occurred while enabling GeyserSkinManager v1.7 (Is it up to date?)
java.lang.NoSuchMethodError: 'void org.yaml.snakeyaml.parser.ParserImpl.<init>(org.yaml.snakeyaml.reader.StreamReader, org.yaml.snakeyaml.LoaderOptions)'
        at com.github.camotoy.geyserskinmanager.relocations.jackson.dataformat.yaml.YAMLParser.<init>(YAMLParser.java:204) ~[?:?]
        at com.github.camotoy.geyserskinmanager.relocations.jackson.dataformat.yaml.YAMLFactory._createParser(YAMLFactory.java:505) ~[?:?]
        at com.github.camotoy.geyserskinmanager.relocations.jackson.dataformat.yaml.YAMLFactory.createParser(YAMLFactory.java:393) ~[?:?]
        at com.github.camotoy.geyserskinmanager.relocations.jackson.dataformat.yaml.YAMLFactory.createParser(YAMLFactory.java:15) ~[?:?]
        at com.github.camotoy.geyserskinmanager.relocations.jackson.databind.ObjectMapper.readValue(ObjectMapper.java:3637) ~[?:?]
        at com.github.camotoy.geyserskinmanager.common.Configuration.create(Configuration.java:46) ~[?:?]
        at com.github.camotoy.geyserskinmanager.spigot.GeyserSkinManager.onEnable(GeyserSkinManager.java:19) ~[?:?]
        at org.bukkit.plugin.java.JavaPlugin.setEnabled(JavaPlugin.java:263) ~[patched_1.14.4.jar:git-Paper-245]
        at org.bukkit.plugin.java.JavaPluginLoader.enablePlugin(JavaPluginLoader.java:338) ~[patched_1.14.4.jar:git-Paper-245]
        at org.bukkit.plugin.SimplePluginManager.enablePlugin(SimplePluginManager.java:420) ~[patched_1.14.4.jar:git-Paper-245]
        at org.bukkit.craftbukkit.v1_14_R1.CraftServer.enablePlugin(CraftServer.java:467) ~[patched_1.14.4.jar:git-Paper-245]
        at org.bukkit.craftbukkit.v1_14_R1.CraftServer.enablePlugins(CraftServer.java:381) ~[patched_1.14.4.jar:git-Paper-245]
        at net.minecraft.server.v1_14_R1.MinecraftServer.a(MinecraftServer.java:474) ~[patched_1.14.4.jar:git-Paper-245]
        at net.minecraft.server.v1_14_R1.DedicatedServer.init(DedicatedServer.java:290) ~[patched_1.14.4.jar:git-Paper-245]
        at net.minecraft.server.v1_14_R1.MinecraftServer.run(MinecraftServer.java:876) ~[patched_1.14.4.jar:git-Paper-245]
        at java.lang.Thread.run(Thread.java:830) [?:?]
[12:36:25 INFO]: [GeyserSkinManager] Disabling GeyserSkinManager v1.7
